system-addon/content-src/components/PreferencesPane/PreferencesPane.jsx

```diff
@@ -76,6 +82,11 @@ class PreferencesPane extends React.Component {
               <PreferencesInput className="showTopSites" prefName="showTopSites" value={prefs.showTopSites} onChange={this.handleChange}
                 titleStringId="settings_pane_topsites_header" descStringId="settings_pane_topsites_body" />
 
+              <div className="options">
+                <PreferencesInput className="showMoreTopSites" prefName="topSitesCount" value={prefs.topSitesCount !== 6} onChange={this.handleChange}
```

Comment:
Also, any thoughts welcome on doing the pref as # of tiles vs # of rows vs boolean (more/less). I figured # of top sites is more future proof. I could hack my pref in about:config to show 3 rows, for example (though it would require a tweak in the feed I think).